extensions/resteasy-server-common/deployment/src/main/java/io/quarkus/resteasy/server/common/deployment/ResteasyServerCommonProcessor.java

```diff
@@ -568,6 +568,8 @@ private static void generateDefaultConstructors(BuildProducer<BytecodeTransforme
         final Set<String> allowedAnnotationPrefixes = new HashSet<>(1 + additionalJaxRsResourceDefiningAnnotations.size());
         allowedAnnotationPrefixes.add(packageName(ResteasyDotNames.PATH));
         allowedAnnotationPrefixes.add("kotlin"); // make sure the annotation that the Kotlin compiler adds don't interfere with creating a default constructor
+        allowedAnnotationPrefixes.add("io.quarkus.security"); // same for the security annotations
+        allowedAnnotationPrefixes.add("javax.annotation.security");
```

Comment:
Can you add `jakarta.annotation.security` too?

Reply:
Good idea, I'll do that tomorrow morning

Reply:
Fixed